content/backend/graphql-scala/1-getting-started.md

```diff
@@ -0,0 +1,212 @@
+---
```

Comment:
Before you go to the other comment - here's a though for you:
maybe instead of the tedious and error prone typing you would prepare a bare bones g8 template with exactly the dummy code you have below, then show users how to get it and guide them through the code as you do now.